src/components/FighterInput.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+class MyFighterInput extends Component {
```

Comment:
I agree it's yours, but you can drop the "My" ;)

Reply:
ok correction and commit done